### PR TITLE
Replace mongodb binary backup to tar archive format

### PR DESCRIPTION
**Pull request by KhurtinDN**

### CODE REVIEW

File: internal/databases/mongo/binary/concurrent_downloader.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+	for _, tarObject := range tarObjects {
+		tarToExtract := internal.NewStorageReaderMaker(tarsFolder, tarObject.GetName())
+		tarsToExtract = append(tarsToExtract, tarToExtract)
+	}
+
+	return tarsToExtract, nil
+}
+
+func IsTarBackupFormat(uploader *internal.Uploader, backupName string) bool {
```
Comment (usernamedt) [inline internal/databases/mongo/binary/concurrent_downloader.go:61]: Maybe store such information in a backup sentinel file instead?

File: internal/databases/mongo/binary/dto.go [hunk 2]
```diff
@@ -2,28 +2,42 @@ package binary
...
 }
 
-type Checksum struct {
-	Algorithm string `json:"Algorithm"`
-	Data      string `json:"Data"`
+type BackupFileMeta struct {
+	Path        string      `json:"Path"`
+	FileMode    os.FileMode `json:"FileMode"`
+	Compression string      `json:"Compression"` // used only for restoring old backups
+	FileSize    int64       `json:"-"`
```
Comment (usernamedt) [inline internal/databases/mongo/binary/dto.go:18]: is it intended?

File: internal/databases/mongo/binary/storage_local.go [hunk 3]
```diff
@@ -120,8 +118,8 @@ func (localStorage *LocalStorage) CreateDirectories(directoryMetas []*BackupDire
 		if err != nil {
 			return err
 		}
-		userName := "mongodb"  // todo: get from sentinel!!!
-		groupName := "mongodb" // todo: get from sentinel!!!
+		userName := "mongodb"
```
Comment (usernamedt) [inline internal/databases/mongo/binary/storage_local.go:121]: Why not get it from sentinel? Maybe introduce a config option? Also, maybe just remove the chown logic completely and run WAL-G from the appropriate user?

File: internal/databases/mongo/binary/dto.go (hunk 2 above)
Comment (KhurtinDN) [reply internal/databases/mongo/binary/dto.go:18]: yes, I want skip it in json, but it used during backup process.

File: internal/databases/mongo/binary/storage_local.go (hunk 3 above)
Comment (KhurtinDN) [reply internal/databases/mongo/binary/storage_local.go:121]: it is legacy, need only to restore from old binary backup without tar archive. I want delete all legacy code.

**Review by usernamedt — APPROVED** [review]

**Review by VGoshev — APPROVED** [review]

**Pull request merged**